Documentation/ApiOverview/ExtensionScanner/Index.rst

```diff
@@ -37,7 +37,9 @@ This module has been featured on the TYPO3 youtube channel:
    .. youtube:: UdIYDZgBrQU
 
 
-Quick Start
+.. index:: Admin tool; Scan Extension Files
```

Comment:
IMHO should be: `Scan extension files` in normal text and the alt text of images. And in the index therefore too, no matter what the guilabel says. Or, if you insist for the index: write `Admin tool; 'Scan Extension Files'`